nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -91,10 +91,10 @@ let
     cd ${webroot}
     sudo=exec
     if [[ "$USER" != nextcloud ]]; then
-      sudo='exec /run/wrappers/bin/sudo -u nextcloud --preserve-env=NEXTCLOUD_CONFIG_DIR --preserve-env=OC_PASS'
+      sudo='exec /run/wrappers/bin/sudo -u nextcloud'
     fi
-    export NEXTCLOUD_CONFIG_DIR="${datadir}/config"
-    $sudo \
+    $sudo env \
```

Comment:
This won't work if you invoke `nextcloud-occ` in a custom systemd unit since those require you to set $PATH explicitly (otherwise it'll be empty).

In other words, I think it makes sense to do `${pkgs.coreutils}/bin/env` here.